ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/SingleEntityVulnerabilitiesTable.tsx

```diff
@@ -1,44 +1,69 @@
 import React from 'react';
 import { Button, ButtonVariant } from '@patternfly/react-core';
-import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import {
+    TableComposable,
+    Thead,
+    Tr,
+    Th,
+    Tbody,
+    Td,
+    ExpandableRowContent,
+} from '@patternfly/react-table';
```

Comment:
Prettier wrapping caused me to notice partial alphabetical order.

Do you mind to reorder here, and also in ImageComponentsTable above?

Reply:
Aha, I've been pretty good about checking these and this one slipped by.